internal/install/stack_version.go

```diff
@@ -6,5 +6,5 @@ package install
 
 const (
 	// DefaultStackVersion is the default version of the stack
-	DefaultStackVersion = "7.13.0-SNAPSHOT"
+	DefaultStackVersion = "8.0.0-SNAPSHOT"
```

Comment:
We prefer to stick to the latest stable version, so still 7.13.0.

Reply:
```suggestion
	DefaultStackVersion = "7.13.0-SNAPSHOT"
```